keras_cv/layers/preprocessing/resizing.py

```diff
@@ -255,24 +302,53 @@ def _resize_with_crop(self, inputs):
         else:
             input_dtype = tf.float32
 
-        def resize_with_crop_to_aspect(x):
+        def resize_with_crop_to_aspect_images(x):
```

Comment:
Instead of making two versions of this method, let's just add `interpolation_method` as a parameter, and call the same method for both masks and images, just with different parameters.

Reply:
I have used the `functool.partial` to take care of this. Let me know if you want it to be implemented in any other way.